doc-QuickStart_OpenStack_Provider_Guide/topics/Further_reading.adoc

```diff
@@ -0,0 +1,5 @@
+== Further reading
+
+For more information on {product-title}, see the https://access.redhat.com/documentation/en/red-hat-cloudforms/version-4.0/[official documentation].
```

Comment:
File removed as part of: https://github.com/ManageIQ/manageiq_docs/pull/184/commits/87068f77677f6a031edab31278c1f5272f15b5e0

Didn't need the file, I was planning to remove it in a subsequent commit.